backend/core/detection.py

```diff
@@ -50,6 +51,26 @@
 log = get_logger()
 hij_log = logging.getLogger("hijack_logger")
 mail_log = logging.getLogger("mail_logger")
+try:
+    hij_log_filter = json.loads(os.getenv("HIJACK_LOG_FILTER"))
+except Exception:
+    log.exception("exception")
+    hij_log_filter = []
+
+
+class HijackLogFilter(logging.Filter):
```

Comment:
shouldn't this be in the util class?

Reply:
you are right, I will check potential transfer of the code there.

Reply:
actually no it is specific to hijack logging in detection.
```
try:
    hij_log_filter = json.loads(os.getenv("HIJACK_LOG_FILTER"))
except Exception:
    log.exception("exception")
    hij_log_filter = []


class HijackLogFilter(logging.Filter):
    def filter(self, rec):
        if not hij_log_filter:
            return True
        for filter_entry in hij_log_filter:
            for filter_entry_key in filter_entry:
                if rec.__dict__[filter_entry_key] == filter_entry[filter_entry_key]:
                    return True
        return False
```

Should it be moved as a whole?

Reply:
Not moving to init, tried a lot of approaches and logging imports seem to mess it up. I will leave it at detection since this is where the respective logs are done (and where the filter should be).